bedrock/mozorg/templates/mozorg/mpl/2.0/includes/structured-data-faq.html

```diff
@@ -0,0 +1,193 @@
+}
+}, {
+"@type": "Question",
+"name": "Does MPL 2.0 require that the MPL 2.0 license notice header be included in every file?",
+"acceptedAnswer": {
+"@type": "Answer",
+"text": "<p>The license notice must be in some way \"attached\" to each file. (Sec. 1.4.) In cases where putting it in the file is impossible or impractical, that requirement can be fulfilled by putting the notice somewhere that a recipient \"would be likely to look for such a notice,\" such as a LICENSE file in the same directory as the file. (Ex. A.) The license notice is as short as possible (3 lines) to make it easy to put in as many different types of files as possible.</p><p>While the license permits putting the header somewhere other than the file itself, individual files often end up being distributed on their own, without the rest of the software they were authored with. As a result, putting the license notice in the file is the surest way to ensure that recipients are always notified.</p><p>For your convenience, Mozilla has produced <a href=\"https://www.mozilla.org/MPL/headers/\">boilerplate headers</a> formatted with several common 'comment' syntaxes, suitable for copying and pasting.</p>"
+}
+}, {
+"@type": "Question",
+"name": "What does \"used\" mean in the definition of Contributor Version (Sec. 1.2)?",
+"acceptedAnswer": {
+"@type": "Answer",
+"text": "<p>“Used” in Section 1.2 means an action taken in the process of creating a Contribution or Modification.</p>"
+}
+}, {
+"@type": "Question",
+"name": "\"Contributor\" means one who creates \"Covered Software\", and \"Covered Software\" includes \"the Executable Form\" of MPL-licensed source code. Does this mean that compiling unmodified code to create an executable makes someone a Contributor?",
+"acceptedAnswer": {
+"@type": "Answer",
+"text": "<p>No. We intend Contributors to be those who have created an addition or change to the program that adds new expression -- copyrightable material -- to pre-existing code.Mere compilation is not an act of authorship it does not create such an addition, and so does not make you a Contributor.</p>"
```

Comment:
nit: Need space between "code." and "Mere"?

Reply:
Fixed